app/components/UI/TransactionElement/TransactionDetails/index.js

```diff
@@ -451,7 +489,6 @@ class TransactionDetails extends PureComponent {
             chainId={chainId}
           />
         </View>
-
         {updatedTransactionDetails.hash &&
           status !== 'cancelled' &&
           rpcBlockExplorer !== NO_RPC_BLOCK_EXPLORER && (
```

Comment:
```suggestion
          rpcBlockExplorer &&  
          rpcBlockExplorer !== NO_RPC_BLOCK_EXPLORER && (
```
i think the nested condition should placed here instead